cmd/kgctl/graph.go

```diff
@@ -18,7 +18,7 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
-	"github.com/squat/kilo/pkg/mesh"
+	"github.com/kilo-io/kilo/pkg/mesh"
```

Comment:
ah wait, go-lint is complaining about the import ordering here. Can we actually separate the imports here to follow our normal import convention? And do the same for the other import in the flannel.go file while we're at it?

Reply:
yes

Reply:
small detail, but can we put it below `github.com/spf13/cobra`? this is the pattern we follow in this repo everywhere else